Immense.RemoteControl.Server/Areas/RemoteControl/Pages/Viewer.cshtml

```diff
@@ -48,6 +48,9 @@
         {
             <div class="logo-frame">
                 <img src="@(Model.LogoUrl)" />
+                <div class="beta-pill beta-pill-logo" style="position: absolute;">
```

Comment:
minor: should refrain from using inline styles where possible since it doesn't promote reusability

Reply:
Agreed.  It will be avoided in the rewrite.